app/javascript/react/components/NavBar.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from "react"
+import { Route, Link } from "react-router-dom"
+import MonstersIndex from "./MonstersIndex"
+import MonsterShow from "./MonsterShow"
+import MonsterForm from "./MonsterForm"
+import SignInTile from "./SignInTile"
+
+const NavBar = props => {
+  const [signedIn, setSignedIn] = useState(null)
+
+const fetchUser = async () => {
+  try {
+    const response = await fetch("/api/v1/users", {
```

Comment:
Not required - but we could make a custom path here instead of using the user index (meant for all users), that would fetch to an endpoint of `api/v1/users/current` for example!